test/verify/check-networking-firewall

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python2
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import parent
+from testlib import *
+import time
+
+
+def wait_unit_state(machine, unit, state):
+
+    def active_state(unit):
+        # don't use `systemctl is-active` here because of https://bugzilla.redhat.com/show_bug.cgi?id=1073481
+        # outputs "ActiveState=state\n"
+        line = machine.execute("systemctl show -p ActiveState {}".format(unit))
```

Comment:
Note: `--value` does not yet exist in current CentOS 7.4. Once that moves to 7.5, we can use it.

Reply:
Ah I didn't know about that. Thank you!